### DISCUSSION TITLE
Support for attaching PDF documents via S3

### DISCUSSION THREAD

**Discussion by dickyj** [outline/outline#1298] [General] [answered]
Hi,
Any chance that your wiki supports attaching PDF files and have those uploaded to some s3 buckets?

**Comment by tommoor** [comment, marked as answer]
Howdy, not currently but there’s a ticket for that here - it’s actually one of the most upvoted feature requests so it seems likely this will reach the top of the roadmap soon

https://github.com/outline/outline/issues/859

**Comment by dickyj** [comment]
Ok thanks for the reply, awesome software.

The challenge is to integrate the ACL into the attachments so that it can only be accessed by the right people.
